src/lib/ui/ClimateSettingForm/ClimateSettingScheduleFormClimateSetting.tsx

```diff
@@ -56,11 +43,41 @@ export function ClimateSettingScheduleFormClimateSetting({
               <InputLabel>{t.climateSetting}</InputLabel>
               <span className='seam-label'>{t.climateSettingSubHeading}</span>
             </div>
-            <FormContent
-              control={control}
-              resetField={resetField}
-              hvacModeSetting={hvacModeSetting}
-            />
+            <FormField>
+              <Controller
+                name='climateSetting'
+                control={control}
+                render={({ field: { value, onChange } }) => {
+                  return (
```

Comment:
Consider keeping the de-nested component below.

Reply:
https://github.com/seamapi/react/pull/545/commits/a320b5dc3cee9d2a89624fb6cd6e5a41d5192818